pkg/booleanpolicy/field_metadata.go

```diff
@@ -895,6 +895,15 @@ func initializeFieldMetadata() FieldMetadata {
 			[]storage.EventSource{storage.EventSource_NODE_EVENT},
 			[]RuntimeFieldType{FileAccess}, negationForbidden,
 		)
+
+		f.registerFieldMetadata(fieldnames.FileOperation,
+			querybuilders.ForFieldLabel(search.FileOperation), nil,
+			func(*validateConfiguration) *regexp.Regexp {
+				return fileOperationRegex
+			},
+			[]storage.EventSource{storage.EventSource_NODE_EVENT},
+			[]RuntimeFieldType{FileAccess}, negationForbidden,
```

Comment:
I think this is a copy-paste error; we should be able to negate file operation (e.g. alert on all operations except for ownership change, etc) I'll fix this.